src/main/java/com/johan/blignaut/entelect/vitalityincubator/task2/helpers/GeoDBMappingHelper.java

```diff
@@ -0,0 +1,63 @@
+package com.johan.blignaut.entelect.vitalityincubator.task2.helpers;
+
+import com.johan.blignaut.entelect.vitalityincubator.restserver.model.GeoDBServer;
+import com.johan.blignaut.entelect.vitalityincubator.restclient.model.GeoDBClient;
+import com.johan.blignaut.entelect.vitalityincubator.restserver.model.MetaDataServer;
+import com.johan.blignaut.entelect.vitalityincubator.restserver.model.CityServer;
+import com.johan.blignaut.entelect.vitalityincubator.restclient.ApiException;
+import com.johan.blignaut.entelect.vitalityincubator.restclient.api.GeodbApi;
+import com.johan.blignaut.entelect.vitalityincubator.restclient.model.CityClient;
+import com.johan.blignaut.entelect.vitalityincubator.restserver.model.LinkServer;
+import com.johan.blignaut.entelect.vitalityincubator.restclient.model.LinkClient;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class GeoDBMappingHelper {
+
+    private final GeodbApi geodbApi;
+
+    @Autowired
+    public GeoDBMappingHelper(GeodbApi geodbApi) {
+        this.geodbApi = geodbApi;
+    }
+
+    public GeoDBServer mapGeoDB() throws ApiException {
+        GeoDBServer geoDBServer = new GeoDBServer();
+        GeoDBClient geodbClient = this.geodbApi.getPlaces();
+        List<CityServer> citiesList = new ArrayList<>();
+        for (CityClient cityClient : geodbClient.getData()) {
```

Comment:
If there is more than 10 or so entries, I'd use Java's Parallel stream to map items in parallel.